app/controllers/catalog_controller.rb

```diff
@@ -61,14 +61,38 @@ def show
     super
   end
 
+  def report_issue_action(_documents)
```

Comment:
@awead - I don't love having this just chilling in `catalog_controller`. It seems like it's just cluttering the catalog controller and should maybe be in a separate file.

The main blacklight project puts `email_action` and `validate_email_params` in `app/controllers/concerns/blacklight/catalog.rb`. 

Should we do something similar? Or do you think it's fine to just leave this here? I defer to your Rails expertise. 🙂